src/common/createplay/utils.js

```diff
@@ -0,0 +1,73 @@
+export const defaultInputFields = {
```

Comment:
These snippets need to be moved to a better-named util file.

Reply:
yup